.test-infra/jenkins/jira_utils/jira_manager.py

```diff
@@ -85,23 +89,31 @@ def run(self, dep_name, dep_latest_version, sdk_type, group_id=None):
               Stop handling the JIRA issue for {1}, {2}""".format(parent_issue.key, dep_name, dep_latest_version))
             return
         logging.info("Found the parent issue {0}. Continuous to create or update the sub-task for {1}".format(parent_issue.key, dep_name))
+
       # creating a new issue/sub-task or updating on the existing issue of the dep
-      summary =  _ISSUE_SUMMARY_PREFIX + dep_name + " " + dep_latest_version
+      summary =  _ISSUE_SUMMARY_PREFIX + dep_name
       issues = self._search_issues(summary)
       issue = None
       for i in issues:
         if i.fields.summary == summary:
           issue = i
           break
+      # Create a new JIRA if no existing one.
       if not issue:
         if sdk_type == 'Java':
           issue = self._create_issue(dep_name, dep_latest_version, is_subtask=True, parent_key=parent_issue.key)
         else:
           issue = self._create_issue(dep_name, dep_latest_version)
         logging.info('Created a new issue {0} of {1} {2}'.format(issue.key, dep_name, dep_latest_version))
+      # Add descriptions in to the opening issue.
       elif issue.fields.status.name == 'Open' or issue.fields.status.name == 'Reopened':
         self._append_descriptions(issue, dep_name, dep_latest_version)
         logging.info('Updated the existing issue {0} of {1} {2}'.format(issue.key, dep_name, dep_latest_version))
+      # Check if we need reopen the issue if it was closed. If so, reopen it then add descriptions.
+      elif self._need_reopen(issue, dep_latest_version):
+        self.jira.reopen_issue(issue)
+        self._append_descriptions(issue, dep_name, dep_latest_version)
```

Comment:
Can this be made an additional comment instead of updating the existing description ? (so that users can see lineage  through comment flow) ?

Reply:
Please ignore this and consider my comment "Regarding "Please review ...

(since it might be hard to track the exact comment that contains the version).